p4include/v1model.p4

```diff
@@ -57,11 +57,8 @@ struct standard_metadata_t {
     @alias("intrinsic_metadata.mcast_grp")     bit<16> mcast_grp;
     @alias("intrinsic_metadata.resubmit_flag") bit<1>  resubmit_flag;
     @alias("intrinsic_metadata.egress_rid")    bit<16> egress_rid;
-}
-
-extern Checksum16 {
-    Checksum16();
-    bit<16> get<D>(in D data);
+    /// Indicates that a verify_checksum() method has failed.
+    bit<1>  checksum_error;
```

Comment:
This does not work for ipv4-over-ipv4 tunnel protocol. Two bits are needed to represent checksum errors in both inner and outer IP headers. Further, why don't we use the `error` type to represent checksum error?